CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ include_directories(SYSTEM ${LIBELF_INCLUDE_DIRS})
 
 find_package(BISON REQUIRED)
 find_package(FLEX REQUIRED)
-bison_target(bison_parser src/parser.yy ${CMAKE_BINARY_DIR}/parser.tab.cc)
+bison_target(bison_parser src/parser.yy ${CMAKE_BINARY_DIR}/parser.tab.cc VERBOSE)
```

Comment:
Let's leave this here. Bison output is super useful when debugging parser issues, and I never remember how to turn it on :)